scripts/release-binary.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -p gitAndTools.hub git -i bash
+project=$(basename $(pwd))
```

Comment:
Btw, I think it would be nice to write a comment about how this file is supposed to be used (run on each CI run in `master`) and what it assumes. AFAIU it assumes presence of `release` file (`release/default.nix`?).

Also it would be nice to add SPDX header to make it easier to use this file in repos that are REUSE compliant. AFAIU, currently it is supposed to be copy-pasted because it is hard to import bash code from another repo.

Reply:
It's not supposed to be copy-pasted, we can just `curl ... | bash` it.

Reply:
Hm, ok, then I don't really care about SPDX header (still makes sense to add it to comply with our guidelines though), but please add some comments about usage. And I suppose after this PR is merged we'll have to make a tag so that people will be able to pass a link pointing to this tag to `curl` (one can also refer to a particular commit, but tags look more neat).

Reply:
Once this is merged, just `curl https://raw.githubusercontent.com/serokell/scratch/master/scripts/release-binary.sh | bash`

Reply:
I'd rather not do it this way because one day someone may move this script into another location and things will stop working for everybody. Let's use something permanent instead of `master`.